src/repository/meatUsers.repostitory.js

```diff
@@ -0,0 +1,150 @@
+const { connection } = require("../config/mysql");
+const {
+  createPlaceholderString,
+  toMysqlTimestampString,
+} = require("../utils/mysql.utils");
+const MeatUser = require("../entities/meatUser.entity");
+const MeatUserStatus = require("../enums/meatUserStatus.enum");
+const MeatUserRole = require("../enums/meatUserRole.enum");
+
```

Comment:
Perhaps you have missed out hasMissingKey check.

Reply:
i just pass the id inside tho. 
I didn't pass in an object and save the entire object